homeassistant/components/system_bridge/coordinator.py

```diff
@@ -145,10 +145,13 @@ async def async_handle_module(
         setattr(self.systembridge_data, module_name, module)
         self.async_set_updated_data(self.systembridge_data)
 
-    async def _listen_for_data(self) -> None:
+    def _listen_for_data(self) -> None:
         """Listen for events from the WebSocket."""
         try:
-            await self.websocket_client.listen(callback=self.async_handle_module)
+            self.hass.async_create_background_task(
+                self.websocket_client.listen(callback=self.async_handle_module),
+                name="System Bridge WebSocket Listener",
+            )
```

Comment:
Since you are no longer awaiting here the exception catch is a effectively a no-op here